src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -187,6 +193,10 @@ export const GithubConfigurationGet = async (req: Request, res: Response, next:
 			log
 		);
 
+		// Sort to that orgs ready to be connected are at the top
+		const isReady = (i: MergedInstallation) => i.isAdmin && !i.isIPBlocked && i.syncStatus !== "FINISHED" && i.syncStatus !== "IN PROGRESS" && i.syncStatus !== "PENDING";
```

Comment:
// todo - would be nice to type the syncstatus's too

Reply:
yeah, I already have that on the interface that uses it.